dashboards/opentelemetry-collector/opentelemetry-collector.json

```diff
@@ -0,0 +1,198 @@
+                        "resource.label.\"cluster\"",
+                        "resource.label.\"namespace\"",
+                        "resource.label.\"job\"",
+                        "resource.label.\"instance\""
+                      ]
+                    }
+                  },
+                  "unitOverride": "",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "Version: ${metric.labels.version}, Instance:${resource.labels.instance}",
+                "minAlignmentPeriod": "60s",
+                "targetAxis": "Y1",
+                "dimensions": [],
+                "measures": [],
+                "breakdowns": []
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            },
+            "chartOptions": {
+              "mode": "COLOR",
+              "showLegend": false,
+              "displayHorizontal": false
+            }
+          },
+          "id": ""
+        }
+      },
+      {
+        "xPos": 24,
+        "yPos": 16,
+        "width": 24,
+        "height": 16,
+        "widget": {
+          "title": "Cloud Monitoring Metric Point Count",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "legendTemplate": "Status: ${metric.labels.status} Instance: ${resource.labels.instance}",
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [
+                        "metric.label.\"status\"",
+                        "resource.label.\"project_id\"",
+                        "resource.label.\"location\"",
+                        "resource.label.\"cluster\"",
+                        "resource.label.\"namespace\"",
+                        "resource.label.\"job\"",
+                        "resource.label.\"instance\""
+                      ],
+                      "perSeriesAligner": "ALIGN_RATE"
+                    },
+                    "filter": "metric.type=\"prometheus.googleapis.com/otelcol_googlecloudmonitoring_point_count_total/counter\" resource.type=\"prometheus_target\""
+                  }
+                }
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
+      },
+      {
+        "yPos": 16,
+        "width": 24,
+        "height": 16,
+        "widget": {
+          "title": "Cloud Observability API Request Count",
```

Comment:
Would this metric include RPCs to other observability backends too? If so could we give this chart a more generic name?

Reply:
This tracks RPCs only to GCP Cloud Observability